### PR TITLE
upgrade to jekyll 4 for more speed

### PR DESCRIPTION
**Pull request by maxandersen**
jekyll 3 ~8sec
jekyll 4 ~5sec

and you don't get pages of deprecation warnings in newer ruby versions.

** If you are updating a guide, please submit your pull request to the main repository: https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc **

### CODE REVIEW

**Comment by gastaldi** [comment]
Build failed, something missing in the Travis CI descriptor? We should really move to Github actions though :stuck_out_tongue:

**Comment by maxandersen** [comment]
> Build failed, something missing in the Travis CI descriptor? We should really move to Github actions though 😛

review https://github.com/quarkusio/quarkusio.github.io/pull/476 and we can move ;)

File: Gemfile [hunk 1]
```diff
@@ -1,5 +1,7 @@
 source "https://rubygems.org"
 
+ruby '2.7.0'
```
Comment (gsmet) [inline Gemfile:3]: Just remove that line and it should work on Travis AFAICS.

Comment (maxandersen) [reply Gemfile]: as long as travis defaults to ruby 2.5+ it should work. force pushed.

**Comment by gastaldi** [comment]
The Gemfile.lock changes when I run `bundle install`, maybe it would be nice to push those changes too

```diff
diff --git a/Gemfile.lock b/Gemfile.lock
index c95ed44a..829638f0 100644
--- a/Gemfile.lock
+++ b/Gemfile.lock
@@ -85,8 +85,5 @@ DEPENDENCIES
   minima (~> 2.0)
   tzinfo-data
 
-RUBY VERSION
-   ruby 2.7.0p0
-
 BUNDLED WITH
-   2.1.2
+   2.1.4
```

**Comment by gastaldi** [comment]
Rebased and added the Gemfile.lock changes proposed in https://github.com/quarkusio/quarkusio.github.io/pull/684#issuecomment-673182351.

I'll merge when the CI passes

**Pull request merged**